azerothcore/azerothcore-wotlk
# fix(DB/quest_poi_points): Fix The Battered Hilt Questline

## CHANGES PROPOSED:
-  Fix the quest poi points for all Horde/Alliance quests as they were completely messed up

**The following quest are hereby fixed:**
- 24555
- 14444
- 24557
- 14457
- 24556
- 20438
- 24800
- 24801
- 24796
- 24461
- 24480

Examples: 
![image](https://user-images.githubusercontent.com/55922592/101165728-704b0d00-3637-11eb-970e-e8eb75daec77.png)

![2020-12-04 11_04_46-Window](https://user-images.githubusercontent.com/55922592/101165892-b1dbb800-3637-11eb-8007-33ca65c1dafd.png)

![2020-12-04 10_56_32-Window](https://user-images.githubusercontent.com/55922592/101165903-b4d6a880-3637-11eb-805a-e21c0891ca38.png)



## ISSUES ADDRESSED:
- None



## TESTS PERFORMED:
- Applied SQL script
- Tested all quests ingame



## HOW TO TEST THE CHANGES:


All ID's from above need to be tested!

1. .q rem & .q a (Pick an quest id from above)
2. Check quest poi point (Open your world map and go to the location)
3. .q c (Quest id which you chose to test)
4. Check quest poi point again (Open your world map and go to the location)

## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
